networkx/algorithms/bipartite/matching.py

```diff
@@ -577,7 +582,11 @@ def minimum_weight_full_matching(G, top_nodes=None, weight="weight"):
     # where edges are missing (as opposed to zeros, which is what one would
     # get by using toarray on the sparse matrix).
     weights_sparse = biadjacency_matrix(
-        G, row_order=U, column_order=V, weight=weight, format="coo"
+        G,
+        row_order=U,
+        column_order=V,
+        weight=weight,
+        format="coo",
```

Comment:
same here.

Reply:
Ditto ^

Reply:
I pip installed the latest black linter package into my env and ran black linter on the code and it caused these changes, I'll reverse these changes.

Reply:
Okay!